relay_messages.go

```diff
@@ -94,14 +95,18 @@ func (cr lazyCallRes) OK() bool {
 type lazyCallReq struct {
 	*Frame
 
-	caller, method, delegate, key, as []byte
-
 	checksumTypeOffset             uint16
 	arg2StartOffset, arg2EndOffset uint16
 	arg3StartOffset                uint16
 
-	checksumType     ChecksumType
-	isArg2Fragmented bool
+	caller, method, delegate, key, as []byte
+	arg2Appends                       []relay.KeyVal
+	checksumType                      ChecksumType
+	isArg2Fragmented                  bool
+
+	// Intentionally an array to combine allocations with that of lazyCallReq
+	// TODO(echung): pool the appends buffers
+	arg2InitialBuf [2]relay.KeyVal
```

Comment:
let's start with `[1]` since we only plan to append a single key-value initially